### PR TITLE
Remove unnecessary message serialization in clob AnteHandle

### PR DESCRIPTION
**Pull request by roy-dydx**
### Changelist
Especially due to message serialization, this is more expensive than it should be. If msg is necessary, it can be serialized in individual logs.

### Test Plan
N/A

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

File: protocol/x/clob/ante/clob.go [hunk 1]
```diff
@@ -63,20 +63,8 @@ func (cd ClobDecorator) AnteHandle(
 	msgs := tx.GetMsgs()
 	var msg = msgs[0]
 
-	// Set request-level logging tags
-	ctx = log.AddPersistentTagsToLogger(ctx,
-		log.Module, log.Clob,
-		log.Callback, lib.TxMode(ctx),
```
Comment (roy-dydx) [inline protocol/x/clob/ante/clob.go:69]: @jonfung-dydx Lmk if there are any specific logs that need Msg. Don't want to serialize it for every CheckTx.

Comment (jonfung-dydx) [reply protocol/x/clob/ante/clob.go:69]: leave the Module=Clob tag

Comment (roy-dydx) [reply protocol/x/clob/ante/clob.go:69]: Leaving that out for now to avoid the hundreds or thousands of allocs per second.

**Comment by roy-dydx** [comment]
@Mergifyio backport release/protocol/v5.x

**Pull request merged**